### ISSUE TITLE
Is this repo going to be updated?

### ISSUE THREAD

**Issue by zbalkan** [AndersRodland/ConfigMgrClientHealth#61]
The web service is rewritten for .Net Framework 4.8 and there are some issues I'm struggling with. Yet, I could not see what's going on. Would yo update the repo, so that we can contribute about the issues with PRs?

**Comment by AndersRodland** [comment]
Yes the code for 2.0.1 will be uploaded today once I'm off work. It has always been the intention to keep it open source.
